students/itsabhi/Lesson4/mailroom2.py

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env python3
+
+def sending_thanksv2(donor_hist):
+
+    donor = input("Type the Full Name of the donor here or type List to display current donors : ")
+    if donor == "list":
+                for k in donor_hist:
+                    print(k)
+                donor = input("Type the Full Name of the donor here: ")
+    if donor in donor_hist.keys():
```

Comment:
The `in` operator checks for existence of keys in dict so no need to call `keys()` here